client/app/components/dialog-editor-boxes/dialog-editor-boxes.component.js

```diff
@@ -0,0 +1,92 @@
+(function() {
+  'use strict';
+
+  angular.module('app.components')
+    .component('dialogEditorBoxes', {
+      controller: function(DialogEditor, DialogEditorModal, lodash) {
+        /**
+         * Load component data
+         */
+        this.$onInit = function() {
+          this.service = DialogEditor;
+          // load data from service
+          this.dialogTabs = DialogEditor.getData().content[0].dialog_tabs;
```

Comment:
@himdel's [comment from previous PR](https://github.com/ManageIQ/manageiq-ui-self_service/pull/37#discussion_r62968802):

> Seeing you're doing that getData().content[0].dialog_tabs in multiple places, maybe that's a bit of an implementation detail, that should be hidden away in the service.
> 
> Maybe even in general, might prefer to move more logic away from the controller into the service..
> 
> EDIT: I guess the same goes for this.dialogTabs[DialogEdit.activeTab].dialog_groups

still not solved